layout/explore/explore-detail/dataset-layers/component.js

```diff
@@ -8,22 +8,51 @@ import DatasetLayerCard from './dataset-layer-card';
 import './styles.scss';
 
 function DatasetLayers(props) {
-  const { layers, dataset } = props;
+  const { layers, dataset, layerGroups } = props;
+  // Flag indicating if this layer group is of "timeline" type
+  const isATimelineDataset = layers.find(l => l.layerConfig.timeline);
```

Comment:
could we create a selector for each of these conditions, and maybe just return a layer to pass in ? 

something like: 

```javascript
activeLayer: selectors.getActiveLayer(state)
```

```javascript
function DatasetLayers(props) {
  const { activeLayer, dataset } = props;
   return <DatasetLayerCard layer={activeLayer} dataset={dataset} />
}
```